wp-content/themes/engage/templates/page-tool.twig

```diff
@@ -1,117 +1,82 @@
 {% extends "base.twig" %}
 
 {% block content %}
-	<article class="container container--lg post-type--{{post.post_type}}" id="post--{{post.ID}} page-tools">
-
+	<article class="container container--md post-type--{{post.post_type}}" id="post--{{post.ID}} page-tools">
+		<header class="article__header">
+			<h1 class="article__title">Tools</h1>
+		</header>
 		<div class="article__content">
-			<article class="quiz-creator-product-page" id="tool-landing-page">
-				<section class="hero aqua" style="margin-top: -36px;">
-					<div class="container container--xl wrapper--pad">
-							<div class="quiz-creator-intro-copy">
-								<h1>Create interactive, embeddable quizzes that are proven to increase engagement online.</h1>
-								<a href="/research/online-polls-and-quizzes/">Read the report →</a>
-
-								<a class="btn btn-primary" href="https://mediaengagement.org/wp-login.php?action=register">Register</a>  Or, <a href="https://mediaengagement.org/wp-login.php">login</a>
+			<section>
+				<div class="row">
+					<div class="col-md-6">
```

Comment:
Do you know if these `col-md-6` and `row` divs actually do anything? If not, could you remove them and do any other similar clean-up?

Reply:
It's looking like these are still being used for some formatting on multiple places.